src/doc/book/unsized-types.md

```diff
@@ -47,14 +47,14 @@ pointers, can use this `impl`.
 # ?Sized
 
 If you want to write a function that accepts a dynamically sized type, you
-can use the special bound, `?Sized`:
+can use the special syntax, `?Sized`:
```

Comment:
I would defer to the book over the actual Sized docs here; I haven't gotten to them yet. I would describe this as a special bound. While it is also technically special syntax, it's like, a sub-syntax: it's still a bound, but `?Sized` is a special case within the trait bound syntax. if that makes sense.